### PR TITLE
Main stories

### PR DESCRIPTION
**Pull request by jbaltres**
create a main component for HomeScreen, and renamed and improved code by comments on PR #3
image

### CODE REVIEW

File: src/components/Textarea.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+const SectionName = styled.div`
+  color: white;
+`;
+
+const SectionGPS = styled.div`
+  color: white;
+`;
+
+function Textarea() {
```
Comment (marwinburesch) [inline src/components/Textarea.js]: I'm not sure if I like the name, textarea sounds more like a parapgraph or something where I would expect a lot of text. Try to connect the name with the parent component it is used with.

File: src/components/Textarea.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import styled from "styled-components";
+
+const Textbox = styled.div`
```
Comment (marwinburesch) [inline src/components/Textarea.js]: This isn't really a `Textbox` more like a wrapper, it defines the positioning of its children and some styling, the actual text will be somewhere else

File: src/components/Counter.js [hunk 3]
```diff
@@ -1,10 +1,58 @@
...
+  color: white;
+`;
 const CounterWrapper = styled.div`
   display: flex;
 `;
 
-const CounterDescription = styled.span`
-  color: "white";
+const CounterArea1 = styled.div`
+  color: ${props => (props.marwin ? "green" : "red")};
```
Comment (marwinburesch) [inline src/components/Counter.js]: ehem...

File: public/index.html [hunk 4]
```diff
@@ -10,6 +10,11 @@
       content="Web site created using create-react-app"
     />
     <link rel="apple-touch-icon" href="logo192.png" />
+    <link
+      rel="stylesheet"
+      href="https://fontlibrary.org/face/segment7"
+      type="text/css"
+    />
```
Comment (marwinburesch) [inline public/index.html]: So you've aded a font library but the component still looks like it's from the 90's

File: src/GlobalStyles.js [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+*,
+*:before,
+*:after {
+  box-sizing: border-box;
+}
+
+body {
+  font-size: 16px;
+  margin: 0;
+  font-family: 'Roboto', sans-serif;
```
Comment (marwinburesch) [inline src/GlobalStyles.js]: I think you might have forgotten to add the relevant stylesheet to your index.html as well

File: src/components/Textarea.js (hunk 2 above)
Comment (oarzani) [reply src/components/Textarea.js]: sehr clever ;)

File: public/index.html (hunk 4 above)
Comment (jbaltres) [reply public/index.html]: I put that in my Backlog, thanks

**Pull request merged**